src/sonic-yang-models/yang-models/sonic-banner.yang

```diff
@@ -0,0 +1,50 @@
+module sonic-banner {
+    yang-version 1.1;
+    namespace "http://github.com/sonic-net/sonic-banner";
+    prefix banner_message;
+    description "BANNER_MESSAGE YANG Module for SONiC-based OS";
+    revision 2023-05-18 {
+        description "First Revision";
+    }
+    container sonic-banner {
+        container BANNER_MESSAGE {
+            description "BANNER_MESSAGE part of config_db.json";
+            container MESSAGE {
+                leaf state {
+                    type string {
```

Comment:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L127
Can you use admin_mode from sonic-types.yang?

Reply:
Done